in_text/text_00_parent.yml

```
@@ -16,22 +16,22 @@ abstract: >-
   <ol>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e">1) Waterbody Information</a> - One shapefile of polylines for 70 river segments in this study, and one shapefile metadata file of reservoir polygons for the Pepacton and Cannonsville reservoirs</li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a287382ce38aaa2449131">2) Model Driver Data </a> - Data used to drive predictive models (daily meteorology for river reaches and reservoirs, observed reservoir diversions and releases)</li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a288982ce38aaa2449133">3) Model Configurations </a> - Model parameters and metadata used to configure GLM 3.1 reservoir models </li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a28a782ce38aaa2449137">4) Model Predictions </a> - Temperature predictions data files, including GLM 3.1 predictions of ourflow and water temperature for reservoir outflow reaches, stream temperature predictions from the distance-weighted-average lotic-lentic input network, and 7-day ahead deep learning water temperature forecasts at 5 priority sites. </li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e">Waterbody Information</a> - One shapefile of polylines for 70 river segments in this study, and one shapefile of reservoir polygons for the Pepacton and Cannonsville reservoirs</li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a287382ce38aaa2449131">Model Driver Data </a> - Data used to drive predictive models (daily meteorology for river reaches and reservoirs, observed reservoir diversions and releases)</li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a288982ce38aaa2449133">Model Configurations </a> - Model parameters and metadata used to configure GLM 3.1 reservoir models </li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a28a782ce38aaa2449137">Model Predictions </a> - Temperature predictions data files, including GLM 3.1 predictions of outflow and water temperature for reservoir outflow reaches, stream temperature predictions from the distance-weighted-average lotic-lentic input network, and 7-day ahead deep learning water temperature forecasts at 5 priority sites. </li>
   <br/>
   <p>This research was funded by the USGS.</p>
 
-build-environment: Multiple computer operating systems were used to generate these data, including Windows, linux, OSX. The open source languages R and Python were used on all systems.
+build-environment: Multiple computer operating systems were used to generate these data, including Windows, Linux, and OSX. The open source languages R and Python were used on all systems.
 
 process-date: !expr format(Sys.time(),'%Y%m%d')
 process-description: >-
     This is the landing page for the other data release sections; processing descriptions can be found on each child item.
 indirect-spatial: U.S.A.
-latitude-res: 0.1
+latitude-res: 0.1 
```

Still curious about how resolution gets chosen for non-gridded data. I know that this was taken from last year's data release, but it would be useful to know where it originated.
```suggestion
latitude-res: 0.1
```